docs/docs-content/clusters/edge/site-deployment/cluster-deployment.md

```diff
@@ -150,73 +150,74 @@ Use the following steps to create a new host cluster so that you can add Edge ho
 
-  <Tabs>
+        <Tabs>
 
-  <TabItem value="calico" label="Calico">
+        <TabItem value="calico" label="Calico">
 
-    In the Calico pack YAML file default template, uncomment `manifests.calico.env.calicoNode.IP_AUTODETECTION_METHOD` and
-    set its value to `kubernetes-internal-ip`. This tells Calico to use the address assigned to the Kubernetes node.
+          In the Calico pack YAML file default template, uncomment `manifests.calico.env.calicoNode.IP_AUTODETECTION_METHOD` and
+          set its value to `kubernetes-internal-ip`. This tells Calico to use the address assigned to the Kubernetes node.
 
-    ```yaml {11}
-    manifests:
-        calico:
-            ...
-            env:
-            # Additional env variables for calico-node
-            calicoNode:
-                #IPV6: "autodetect"
-                #FELIX_IPV6SUPPORT: "true"
-                #CALICO_IPV6POOL_NAT_OUTGOING: "true"
-                #CALICO_IPV4POOL_CIDR: "192.168.0.0/16"
-                IP_AUTODETECTION_METHOD: "kubernetes-internal-ip"
-    ```
+          ```yaml {11}
+          manifests:
+              calico:
+                  ...
+                  env:
+                  # Additional env variables for calico-node
+                  calicoNode:
+                      #IPV6: "autodetect"
+                      #FELIX_IPV6SUPPORT: "true"
+                      #CALICO_IPV6POOL_NAT_OUTGOING: "true"
+                      #CALICO_IPV4POOL_CIDR: "192.168.0.0/16"
+                      IP_AUTODETECTION_METHOD: "kubernetes-internal-ip"
+          ```
 
-  </TabItem>
+        </TabItem>
 
-  <TabItem value="flannel" label="Flannel">
+        <TabItem value="flannel" label="Flannel">
 
-    In the Flannel pack YAML file, add a line `- "--iface=INTERFACE_NAME"` in the default template under
-    `charts.flannel.args`. Replace `INTERFACE_NAME` with the name of the NIC. For example, add the line `- "--iface=eno32`
-    if the NIC name of your control plane nodes is `eno32`.
+          In the Flannel pack YAML file, add a line `- "--iface=INTERFACE_NAME"` in the default template under
+          `charts.flannel.args`. Replace `INTERFACE_NAME` with the name of the NIC. For example, add the line `- "--iface=eno32`
+          if the NIC name of your control plane nodes is `eno32`.
 
-    ```yaml {8}
-    charts:
-        flannel:
-            ...
-            # flannel command arguments
-            args:
-            - "--ip-masq"
-            - "--kube-subnet-mgr"
-            - "--iface=eno32"
-    ```
+          ```yaml {8}
+          charts:
+              flannel:
+                  ...
+                  # flannel command arguments
+                  args:
+                  - "--ip-masq"
+                  - "--kube-subnet-mgr"
+                  - "--iface=eno32"
+          ```
 
-  </TabItem>
-  
-  <TabItem value="cilium" label="Cilium">
-    You do not need to make any adjustments to the Cilium pack.
-  </TabItem>
+        </TabItem>
```

Comment:
Like with the OpenStack comment, the tabs and contained content only needed a couple of spaces per line to correct the indentation.

Like before, it doesn't matter now as the rendered indentation looks good.